programs/survival/single_bedrock.sc

```diff
@@ -0,0 +1,82 @@
+global_DIM = 'the_nether';
+global_CHUNK_SIZE = 16;
+global_DRAW_RADIUS = 1;
+global_INTERVAL = 20;
+global_DURATION = global_INTERVAL + 2;
+global_TOP_Y = 127;
+global_CHECK_HEIGHT = 4;
+global_BLOCK_NAME = 'bedrock';
+global_DRAW_HEIGHT = 6;
+global_COLOR = 0x00ff0050;
+
+__config() -> {
+  'command_permission' -> 'players'
+};
+
+global_enabled = false;
+global_player = null;
+global_from_bottom = false;
+global_last_chunk_x = null;
+global_last_chunk_z = null;
+global_shapes = null;
+
+__command() -> (
+  global_player = player();
+  global_enabled = !global_enabled;
+  _clear_cache();
+  if(global_enabled, (
+    in_dimension(global_DIM, schedule(0, '_tick'));
+    print('Enabled');
```

Comment:
Do you think this could be replaced by an actionbar message? Cos that way it only goes to the player that's using it, and no one else.

Reply:
Looks like other players would not see that message. I tested using LAN multiplayer.

Reply:
wait, do you want other players to be able to see this message?

Reply:
No, I don't.

Reply:
Print without arguments only goes to the executing source.